### PR TITLE
feat: add reference to sea orm cli for newcomers

### PR DESCRIPTION
**Pull request by EstebanBorai**
For people who is reading the code the first time it would be helpful
to have a quick access to the CLI documentation at this point.

## PR Info

- Closes 

- Dependencies:
  - 

- Dependents:
  - 

## Adds

A reference to the CLI installation page for the Migrations section.

### CODE REVIEW

**Review by billy1624 — COMMENTED** [review]
Hey @EstebanBorai, thanks for contributing!!

Could you please add the lines to... as well?
- https://github.com/SeaQL/seaql.github.io/edit/master/SeaORM/docs/05-migration/01-setting-up-migration.md
- https://github.com/SeaQL/seaql.github.io/edit/master/SeaORM/versioned_docs/version-0.6.x/05-migration/01-setting-up-migration.md

**Comment by EstebanBorai** [comment]
> Hey @EstebanBorai, thanks for contributing!!
> 
> Could you please add the lines to... as well?
> 
> * https://github.com/SeaQL/seaql.github.io/edit/master/SeaORM/docs/05-migration/01-setting-up-migration.md
> * https://github.com/SeaQL/seaql.github.io/edit/master/SeaORM/versioned_docs/version-0.6.x/05-migration/01-setting-up-migration.md

Of course! Now we have the comment for versions `0.7.x` and `0.6.x`. Thanks for suggesting!

**Review by billy1624 — COMMENTED** [review]
Got an idea! Wouldn't it be better to use anchor tag directly?
- https://github.com/EstebanBorai/seaql.github.io/compare/patch-1...billy1624:pr/29

**Comment by EstebanBorai** [comment]
@billy1624 I replaced instances of the link with the anchor one. Also noticed I was missing the ref to the `[1]` markdown variable.

File: SeaORM/docs/05-migration/01-setting-up-migration.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ pub struct Model {
 
 Setup the migration directory by executing `sea-orm-cli migrate init`.
 
+> If you don't have `sea-orm-cli` installed yet, please refer to <a href="https://www.sea-ql.org/SeaORM/docs/generate-entity/sea-orm-cli/">Using sea-orm-cli</a>
+ page to learn more about the Sea ORM CLI.
```
[the remark this answers is not in the document]
Comment (EstebanBorai) [reply SeaORM/docs/05-migration/01-setting-up-migration.md:24]: So, to be clear:

1. Remove anchor HTML tags and go back to MD links
2. Use relative links instead

Both changes, correct?

[the remark this answers is not in the document]
Comment (billy1624) [reply SeaORM/docs/05-migration/01-setting-up-migration.md:24]: Sorry for the delay. Correct! Basically, replace the lines with the following :)

```md
> If you don't have `sea-orm-cli` installed yet, please refer to [Using sea-orm-cli](03-generate-entity/01-sea-orm-cli.md) page to learn more about the Sea ORM CLI.
```

**Comment by billy1624** [comment]
Hey @EstebanBorai, I have added a paragraph that teach newcomers to install `sea-orm-cli` at the beginning of the migration docs.
- https://github.com/SeaQL/seaql.github.io/pull/38/files#diff-1719b578922041c6bc6e15fc64c8b1499e88d48c67e6c855c0bbfa92ecdc7183R11-R17

**Comment by EstebanBorai** [comment]
Thanks for your time and work on documentation @billy1624! I'm closing this in favor of: https://github.com/SeaQL/seaql.github.io/pull/38/files#diff-1719b578922041c6bc6e15fc64c8b1499e88d48c67e6c855c0bbfa92ecdc7183R11-R17

**Comment by billy1624** [comment]
Thanks! @EstebanBorai

**Pull request closed without merging**